Core/src/org/sleuthkit/autopsy/report/infrastructure/ReportVisualPanel1.java

```diff
@@ -67,9 +75,12 @@ public ReportVisualPanel1(ReportWizardPanel1 wizPanel, Map<String, ReportModuleC
 
     // Initialize the list of ReportModules
     private void initModules() {
+
+        WindowManager.getDefault().getMainWindow().setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR));
```

Comment:
Always setting the cursor using the main window is not working in all settings. Specifically, as in the demo today, the wait cursor does not appear when you go to the wizard via the command line ingest options panel. Have you tried just getting this panel's parent?